docker/Dockerfile

```diff
@@ -1,19 +1,59 @@
 FROM continuumio/anaconda3
 
-WORKDIR /usr/src/project
-COPY . /usr/src/project
-
 RUN apt-get update && apt-get upgrade -y \
-	
 	&& apt-get install -y \
-	 	libpq-dev \
-	 	build-essential \
-	 	git \
+		libpq-dev \
+		build-essential \
+		git \
+		sudo \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN conda install -y -c conda-forge \
+		tensorflow=1.0.0 \
```

Comment:
I think 1.3.0 is available, it's probably best to use the latest version, instead of 1.0.0.

Reply:
I didnt't get to reading/running the TF part yet and just assumed that StevenBunkley pinned that version on purpose. If you know the code will run on 1.3 the "=1.0.0" can be easily dropped. It should surely ease keeping all the other dependencies current.

Reply:
I was just following the installation instructions on the main readme.